src/checks/series/monotonic_decrease_check.rs

```diff
@@ -0,0 +1,227 @@
+use crate::{DataCache, Error, Flag, Timeseries};
+
+/// Timeseries check that detects if a timeseries is monotonically slightly decreasing in a given
+/// window. Useful for detecting evaporation/leakage of a bucket measuring precipitation.
+///
+/// Takes a series of data points representing the window to check for increases, and upper and
+/// lower limits that the total difference over the whole series must be within to meet the
+/// failure condition
```

Comment:
```suggestion
/// The total difference over the whole series must be within the
/// provided upper and lower limits to meet the failure condition.
```
Maybe also worth mentioning that "historically upper and lower limits were used to differentiate if for example the collection bucket was manually emptied (big decrease, but not problematic)", or something like that?

Reply:
Done.